### ISSUE TITLE
Transparent Icon Button / Slider ?

### ISSUE THREAD

**Issue by mokun** [mgarin/weblaf#639]
Hi,

How do I get rid of the pale pink background that comes with the WebButton ? 

![image](https://user-images.githubusercontent.com/1168584/89749772-03a37580-da7e-11ea-8b4a-1c909c53b4d1.png)

I'd like to create button that has a transparent background so that whatever icon image inside the button would appear to "float" on top of background panel.

Can it be done using weblaf and how ?

See below. I saw in the demo that there is a class called IconButton. 

![image](https://user-images.githubusercontent.com/1168584/89749656-7fe98900-da7d-11ea-8a4f-1f3a36ec1ac6.png)

I guess the fourth row of button are NOT transparent. But at least, it appears so. 

When I hover my mouse pointer over it, the pink background color shows up. 

Without the mouse hovering over it, it doesn't show.

**Comment by mgarin** [comment]
You can use basic `StyleId.buttonUndecorated` style:
```
WebButton undecorated = new WebButton ( StyleId.buttonUndecorated );
```
You can provide that style in `WebButton` constructor or set it later at any point in runtime.

You can also apply style to `JButton` if you don't want any direct WebLaF dependencies: 
https://github.com/mgarin/weblaf/wiki/How-to-style-Swing-components

P.S. It's not pink - it's light gray:
```xml
<GradientBackground>
    <color>white</color>
    <color>223,223,223</color>
</GradientBackground>
```
:)

**Comment by mokun** [comment]
```

WebButton undecorated = new WebButton ( StyleId.buttonUndecorated );
..

```

The `StyleId.buttonUndecorated` work well. See below.

![image](https://user-images.githubusercontent.com/1168584/89923931-1e790580-dbb6-11ea-9c9c-396ddfef8151.png)

Question, is there a weblaf enhancement of the plain ol' JSlider in Swing ?

Any quick tips on replacing the color scheme, especially, making it transparent and floating, just like the WebButtton above ?

**Comment by mgarin** [comment]
I haven't fully moved slider to new styling system, so it would be quite hard to change it looks right now. There are some settings you can modify in custom `slider` style coming from `SliderPainter` class, but those won't help in this particular case.

You can follow #493 to get updates on the slider changes once they go live.

**Comment by mgarin** [comment]
I'll close this question as it should now be answered.
Adjusting `slider` view will become possible once  #493 is implemented (possibly in v1.2.15 update).
